src/redux/books/book.js

```diff
@@ -1,25 +1,57 @@
+import { v4 as uuidv4 } from 'uuid';
+
 const ADD_BOOK = 'bookstore/book/ADD_BOOK';
 const DELETE_BOOK = 'bookstore/book/DELETE_BOOK';
 
-const initialState = [];
+const initialState = [
+  {
+    id: uuidv4(),
+    category: 'Fiction',
+    title: 'Harry Potter and the Chamber of Secrets',
+    author: 'J.K. Rowling',
+    progress: 75,
+    currentChapter: 'Chapter 7',
+  },
+  {
+    id: uuidv4(),
+    category: 'Fiction',
+    title: 'The Hobbit',
+    author: 'J.R.R. Tolkien',
+    progress: 75,
+    currentChapter: 'Chapter 1',
+  },
+  {
+    id: uuidv4(),
+    category: 'Action',
+    title: 'The Hunger Games',
+    author: 'Suzanne Collins',
+    progress: 75,
+    currentChapter: 'Chapter 13',
+  },
+];
 
 const bookReducer = (state = initialState, action) => {
   switch (action.type) {
     case ADD_BOOK:
-      return [...state, action.book];
+      return [...state, action.payload];
```

Comment:
- [ ] [_OPTIONAL_]Please make sure that when we add new data through the form all the keys have value since we will use those when we display.

![image](https://user-images.githubusercontent.com/59416136/167181368-019a16bb-21b6-4b55-bb76-9ae0879a3963.png)

![image](https://user-images.githubusercontent.com/59416136/167181425-e2daca07-0bd9-442f-9967-535ec92fce9f.png)